js/main.js

```diff
@@ -50,21 +33,13 @@ const _minute = _second * 60;
 const _hour = _minute * 60;
 const _day = _hour * 24;
 
-// setInterval allows you to do certain action every period of time (like cronjob)
 setInterval(function() {
 
   // Check is current time + 10 minutes smaller than the starting time
-  if(now+1000*60*10 < start) {
-    // If it is, show the box and start counting down
+  if(now + _minute * 10 < start) {
     $('.countdown').show(400);
   } else {
-    liveActive = true;
-    // Otherwise, show the livestream and abjust the height of it to be in
-    // 19:6 aspect ratio
-    // When done hiding the countdown show the livestream, make it take 400ms so there is a smooth transition
```

Comment:
Comments are superfluous since they are describing meta data about the video. 

They are also in the wrong place, since your describing behaviour of `showLive()` in an area where it's called.

Do you propose that in the future if you were to have two calls to show live you'd also have this comment in two places?